config/manager/manager.yaml

```diff
@@ -11,16 +11,17 @@ metadata:
   name: controller-manager
   namespace: system
   labels:
-    control-plane: controller-manager
+    app.kubernetes.io/component: controller-manager
 spec:
   selector:
     matchLabels:
-      control-plane: controller-manager
+      app.kubernetes.io/component: controller-manager
```

Comment:
heads up: updates to the label selector of running deployments are forbidden, a new deployment is needed in that case. Unfortunately OLM doesn't deal with this, in the past at least. When updating the operator, it tries the forbidden label selector update and fails, instead of replacing the deployment.
So either please test if this is still the case, or rename the deployment.

Reply:
Thanks, I think I'll stick only to the DS labels in this PR

Reply:
Similar issue as for FAR PR https://github.com/medik8s/fence-agents-remediation/pull/46

Reply:
I'd suggest to test an update also for just the ds change, I wouldn't be that much surprised if we run into the same issue with our own code similar to OLM

Reply:
Indeed, same issue occured.
tested again successfully after [this](https://github.com/medik8s/self-node-remediation/pull/133/commits/be01bcb8b67a5aa2bcdad86eda39343f898b7702) commit (removing old ds)